src/matrix/room/timeline/entries/reply.js

```diff
@@ -51,6 +51,9 @@ function _createReplyContent(targetId, msgtype, body, formattedBody) {
 }
 
 export function createReplyContent(entry, msgtype, body) {
+    if (entry.isThread) {
+        return createThreadContent(entry, msgtype, body);
```

Comment:
Perhaps it would be good to _also_ include the reply content here? [This is how MSC 3440](https://github.com/matrix-org/matrix-doc/blob/gsouquet/threading-via-relations/proposals/3440-threading-via-relations.md#quote-replies-in-a-thread) seem to do it.

Reply:
I'm not sure if we always need to do that(?). Replies to replies in a thread should probably also be replies. But otherwise we just send a thread message.

Reply:
hmm.. not sure about this, maybe we should always include reply content :thinking:

Reply:
Yeah you're right, we should always include the reply content as well 👍